tg/Network/NetworkStruct.swift

```diff
@@ -7,14 +7,14 @@
 
 import Foundation
 
-struct FromServer: Codable {
+struct GamesManager: Codable {
```

Comment:
GamesManager is inappropriate to name.
For example, NetworkManager is responsible for network logic. It's not a model
GamesResult and GamesModel are better to explain how it will be used